spec/circleci_spec.rb

```diff
@@ -20,6 +21,15 @@
       CircleCi.config.token.should eql 'new-key'
     end
 
+    it 'can access the proxy after being set' do
+      CircleCi.configure do |config|
+        config.proxy = 'http://username:password@myproxy.com:1234'
+      end
+      CircleCi.config.proxy.should eql 'http://username:password@myproxy.com:1234'
+      CircleCi.config.proxy = 'http://myproxy.com'
```

Comment:
Looks like this tests the same thing twice. Lets remove the second testing of setting the proxy on the config.